docs/landing-zone-v2/README.md

```diff
@@ -258,7 +258,7 @@ The following instructions in this section 1 for the boostrap of the Config Cont
 1. Enable the required services to provision the config controller instance.
 
     ```shell
-    gcloud services enable krmapihosting.googleapis.com container.googleapis.com cloudresourcemanager.googleapis.com cloudbilling.googleapis.com serviceusage.googleapis.com servicedirectory.googleapis.com dns.googleapis.com
```

Comment:
just thinking out loud, would it make sense to also add it in the YAML manifests?  that way when new "features" like this pop up, it can handled through code instead of only bootstrap (which only happens once)

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/solutions/experimentation/core-landing-zone/mgmt-project/services.yaml

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/solutions/core-landing-zone/mgmt-project/services.yaml

Reply:
That's fair, it is only required for Policy Controller and Config Management and not the core functionality of KCC so it wouldn't impact the bootstrap process if we moved that to the manifests.

Reply:
either way, it can't hurt to have it in both places

Reply:
will you include the changes mentionned by alain in this PR ?

Reply:
@cartyc , we need this change before our code freeze(friday november 24th). just let me know if you don't have the cycles and I will clone your branche and push the additional change

Reply:
Should have it done tomorrow morning.

Reply:
Changes pushed. @davelanglois-ssc ready for re-review.

Reply:
so you removed it from the readme.md and added it to core-landing-zone package.
can you also add it to the experimentation/core-landing-zone ?

minor but I had no issue with the api also being enabled during bootstrap using gcloud

Reply:
Updated experimentation/core-landing-zone. I read added the command to the readme for now. I think in a future iteration of  this should be marked as optional as it's just required to view the config-sync data in the ui.